jenkinsfiles/ginkgo-gke.Jenkinsfile

```diff
@@ -126,16 +126,33 @@ pipeline {
                 CONTAINER_RUNTIME=setIfLabel("area/containerd", "containerd", "docker")
                 KUBECONFIG="${TESTDIR}/gke/gke-kubeconfig"
                 CNI_INTEGRATION="gke"
+                CILIUM_IMAGE = """${sh(
+                        returnStdout: true,
+                        script: 'echo -n $(${TESTDIR}/print-node-ip.sh)/cilium/cilium:${TAG}'
+                        )}"""
+                CILIUM_OPERATOR_IMAGE= """${sh(
+                        returnStdout: true,
+                        script: 'echo -n $(${TESTDIR}/print-node-ip.sh)/cilium/operator:${TAG}'
+                        )}"""
+                K8S_VERSION= """${sh(
+                        returnStdout: true,
+                        script: 'cd ${TESTDIR}; gke/get-cluster-version.sh'
+                        )}"""
+                FOCUS= """${sh(
+                        returnStdout: true,
+                        script: 'echo ${ghprbCommentBody} | sed -r "s/([^ ]* |^[^ ]*$)//" | sed "s/^$/K8s*/" | tr -d \'\n\''
+                        )}"""
             }
             steps {
                 dir("${TESTDIR}"){
-                    sh 'K8S_VERSION=$(${TESTDIR}/gke/get-cluster-version.sh) CILIUM_IMAGE=$(./print-node-ip.sh)/cilium/cilium:latest CILIUM_OPERATOR_IMAGE=$(./print-node-ip.sh)/cilium/operator:latest ginkgo --focus="$(echo ${ghprbCommentBody} | sed -r "s/([^ ]* |^[^ ]*$)//" | sed "s/^$/K8s*/")" -v --failFast=${FAILFAST} -- -cilium.provision=false -cilium.timeout=${GINKGO_TIMEOUT} -cilium.kubeconfig=${TESTDIR}/gke/gke-kubeconfig -cilium.passCLIEnvironment=true -cilium.registry=$(./print-node-ip.sh)'
+                    sh 'env'
+                    sh 'ginkgo --focus="${FOCUS}" -v -- -cilium.provision=false -cilium.timeout=${GINKGO_TIMEOUT} -cilium.kubeconfig=${KUBECONFIG} -cilium.passCLIEnvironment=true -cilium.registry=$(./print-node-ip.sh) -cilium.image=${CILIUM_IMAGE} -cilium.operator-image=${CILIUM_OPERATOR_IMAGE} -cilium.holdEnvironment=false'
                 }
             }
             post {
                 always {
                     sh 'cd ${TESTDIR}; ./archive_test_results_eks.sh || true'
-                    archiveArtifacts artifacts: 'src/github.com/cilium/cilium/*.zip'
+                    archiveArtifacts artifacts: '**/*.zip'
```

Comment:
Out of curiosity, why did this change? If this was to fix collecting some artefacts then we might want to do it for all the build, and in a different commit? (or PR even).